scripts/f5-config.py

```diff
@@ -252,7 +278,7 @@
         'make_public': True,
         'hosts': []
     },
-    'nova_console': {
+    'nova_spice_console': {
```

Comment:
Why is this being changed? It was previously changed from nova_spice_console to nova_console.